tiled/access_control/access_policies.py

```diff
@@ -360,3 +367,54 @@ async def filters(self, node, principal, authn_access_tags, authn_scopes, scopes
 
         queries.append(query_filter(identifier, tag_list))
         return queries
+
+
+class Data(BaseModel):
+    token: str
+    audience: str
+    actions: list[str]
+    node: List[str]
+
+
+class Input(BaseModel):
+    input: Data
+
+
+class Decision(BaseModel):
+    result: bool
+
+
+class ExternalPolicyDecisionPoint:
```

Comment:
Extract these changes (the ExternalPolicy access_policy and OPA changes) into their own PR. That PR can be into this branch if it's required for merging this branch into main, but the external AuthN and the external AuthZ are orthogonal to each other and are making it harder to review this